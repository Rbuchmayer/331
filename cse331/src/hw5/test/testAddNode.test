CreateGraph g
AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4
AddNode g n1
AddNode g n1
AddNode g n2
CreateGraph h
AddNode h n1
AddNode h n3
AddNode h node