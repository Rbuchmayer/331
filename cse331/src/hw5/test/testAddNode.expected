created graph g
added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g
added node n1 to g
added node n1 to g
added node n2 to g
created graph h
added node n1 to h
added node n3 to h
added node node to h